src/control/cmd/daos/README.md

```diff
@@ -152,4 +150,4 @@ func (cmd *containerScrubCmd) Execute(_ []string) error {
 }
 ```
 
-There are more advanced possibilities here using the go-flags package. For example, we could change the Level argument from a simple string into a custom type that implements the `flags.Completer` interface in order to allow for tab-completion of valid scrub level names.
\ No newline at end of file
+There are more advanced possibilities here using the go-flags package. For example, we could change the Level argument from a simple string into a custom type that implements the `flags.Completer` interface in order to allow for tab-completion of valid scrub level names.
```

Comment:
Nothing has been changed in this line but for some reason Git diff shows this. May be because I used Linux to modify this file?